src/Servers/IIS/AspNetCoreModuleV2/CommonLib/HostFxrResolver.cpp

```diff
@@ -144,6 +186,34 @@ HostFxrResolver::GetHostFxrParameters(
     }
 }
 
+BOOL
+HostFxrResolver::TryGetHostFxrPath(
+    fs::path& hostFxrDllPath,
+    const fs::path& dotnetRoot,
+    const fs::path& applicationPath)
+{
+    std::wstring hostfxrPath;
+    size_t size = MAX_PATH * 2;
+    hostfxrPath.resize(size);
+
+    get_hostfxr_parameters params;
+    params.size = sizeof(get_hostfxr_parameters);
```

Comment:
Is info the right level for this?

Reply:
I don't think the info level matters much for native code, basically the only way you get anything is to turn on ancm debug logs, which will write this to the file, the logging is pretty verbose as this is pretty much all we are going to get later on if people have issues with this code, so I think its important we get the parameters here, otherwise we will be left with the extremely verbose corehost tracing only